src/main/java/com/park/easyrecruit/servlet/interview/Interview.java

```diff
@@ -0,0 +1,25 @@
+package com.park.easyrecruit.servlet.interview;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ *
+ * @author Dragos
+ */
+@WebServlet(name = "Interview", urlPatterns = {"/Interview"})
```

Comment:
Please add servlet guard
```suggestion
@WebServlet(name = "Interview", urlPatterns = {"/Interview"})
@ServletSecurity(value = @HttpConstraint(rolesAllowed = {"*role_name*"}))
```